<ui version="4.0" >
 <class>SvnSwitchDialog</class>
 <widget class="QDialog" name="SvnSwitchDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Subversion Switch</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="2" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="TextLabel1" >
     <property name="text" >
      <string>Tag Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="tagCombo" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Enter the name of the tag</string>
     </property>
     <property name="whatsThis" >
      <string>&lt;b>Tag Name&lt;/b>
&lt;p>Enter the name of the tag to be switched to.
In order to switch to the trunk version leave it empty.&lt;/p></string>
     </property>
     <property name="editable" >
      <bool>true</bool>
     </property>
     <property name="duplicatesEnabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QGroupBox" name="TagTypeGroup" >
     <property name="title" >
      <string>Tag Type</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QRadioButton" name="regularButton" >
        <property name="toolTip" >
         <string>Select for a regular tag</string>
        </property>
        <property name="whatsThis" >
         <string>&lt;b>Regular Tag&lt;/b>
&lt;p>Select this entry for a regular tag.&lt;/p></string>
        </property>
        <property name="text" >
         <string>Regular Tag</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="branchButton" >
        <property name="toolTip" >
         <string>Select for a branch tag</string>
        </property>
        <property name="whatsThis" >
         <string>&lt;b>Branch Tag&lt;/b>
&lt;p>Select this entry for a branch tag.&lt;/p></string>
        </property>
        <property name="text" >
         <string>Branch Tag</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>tagCombo</tabstop>
  <tabstop>regularButton</tabstop>
  <tabstop>branchButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SvnSwitchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>21</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel" >
     <x>21</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SvnSwitchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>67</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel" >
     <x>67</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
